Wine_project_Tanya_new.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>this needs to be a markdown cell, otherwise the ! indicates that you are trying to run bash (hence the /bin/sh error)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-TataAndBigData/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>